pyartcd/pyartcd/pipelines/update_golang.py

```
@@ -658,6 +668,130 @@ def get_stream(stream_name):
+
+        if self.skip_pr:
+            _LOGGER.info("Skipping layered branch PRs (--skip-pr flag set) for branches: %s", sync_branches)
+            return
+
+        results = await asyncio.gather(
+            *[self._update_layered_branch(b, go_version, replacements) for b in sync_branches],
+            return_exceptions=True,
+        )
+
+        summary_lines = []
+        for branch_name, result in zip(sync_branches, results):
```

Ruff (0.15.10)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)